src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -730,4 +730,10 @@ Do you want to proceed?</value>
   <data name="TheRenameTrackingSessionWasCancelledAndIsNoLongerAvailable" xml:space="preserve">
     <value>The rename tracking session was cancelled and is no longer available.</value>
   </data>
+  <data name="Rename2" xml:space="preserve">
+    <value>Rename '{0}'</value>
+  </data>
+  <data name="SingleQuote" xml:space="preserve">
```

Comment:
Do we have any RTL localized languages? If so, what is "Left" and "Right"?

It might be better to still have a single string, and then split it into left/right on the {0} token. Then localizers don't have to understand this relationship at all.